img/l/ligabank.de.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8.89 9.08"><path fill="#fff" fill-rule="evenodd" d="M0 0h8.9v9.08H0z"/><path fill="#00549e" d="M0 0h3.83a5.2 5.2 0 0 0-1.32.94c-.62.63-.98 1.3-.99 2.16 0 .81.45 1.61 1 2.14a6 6 0 0 0 2.14 1.3 9.73 9.73 0 0 0 3.3.57l.93-.04v2.01H0zm8.89 0v6.17l-.93.05c-.48 0-1.11-.04-1.77-.16a6.1 6.1 0 0 1-3.07-1.44s-.71-.64-.7-1.53c0 0-.08-.77.71-1.55 0 0 .57-.6 1.61-1C4.74.53 5.8.08 7 0h1.89zm-3.5 4.99a1.03 1.03 0 1 0 2.07 0 1.03 1.03 0 0 0-2.07 0"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8.89 9.08"><path fill="#fff" d="M0 0h8.9v9.08H0z"/><path fill="#00549e" d="M0 0h3.83a5.2 5.2 0 0 0-1.32.94c-.62.63-.98 1.3-.99 2.16 0 .81.45 1.61 1 2.14a6 6 0 0 0 2.14 1.3 9.73 9.73 0 0 0 3.3.57l.93-.04v2.01H0zm8.89 0v6.17l-.93.05c-.48 0-1.11-.04-1.77-.16a6.1 6.1 0 0 1-3.07-1.44s-.71-.64-.7-1.53c0 0-.08-.77.71-1.55 0 0 .57-.6 1.61-1C4.74.53 5.8.08 7 0h1.89zm-3.5 4.99a1.03 1.03 0 1 0 2.07 0 1.03 1.03 0 0 0-2.07 0"/></svg>
```

Reply:
I don't think the warning about the `fill-rule` attribute is generally helpful. In this case, it doesn't make a difference, but you can't just blindly remove it without checking the result - so it's definitely not always "unnecessary".

Also, if the proposed tools (`svgo` on the command line or `SVGOMG` on the web) don't remove it by themselves either, therefore requiring manual editing, it's rather silly anyway.